javascript/javascript1/week4/creditCardNo/cc.js

```diff
@@ -0,0 +1,13 @@
+function formatCreditCardNumber(number) {
+    number = number.toString();
```

Comment:
super confusing to use `number` both as a string and as a number in the same line. JavaScript allows it but you shouldn't do it anyway. If you're changing to a different type (like `toString()`), create another variable to contain the result.